environments/auth/auth-with-federated/README.md

```diff
@@ -0,0 +1,16 @@
+# Auth with Federated
```

Comment:
I might be blind...but why do I see this file exist in `main`? https://github.com/aws-amplify/amplify-ui/blob/main/environments/auth/auth-with-federated/README.md

If so, it should be modify, instead of showing a new all-green file, right?

And looks like the only differences are 1) title changed from `Facebook` to `Federated`, and no "internal" instructions?

Reply:
Oh yeah, I should've been clear. For context, we previously deleted `auth-with-federated` in https://github.com/aws-amplify/amplify-ui/pull/2596, because that environment didn't do anything in terms of testing. That deletion went to `split-environments/main` feature branch.

But with what happened with federated sign in, I'm re-adding this to the feature branch,.

Reply:
tldr; we removed it in the feature branch, and now are adding it back after what has happened with federatedSignIn failure.

Reply:
Oh, I see. I didn't notice that we are merging into a feature branch 😂